src/data/lovelace.ts

```diff
@@ -20,6 +21,9 @@ export interface LovelaceViewConfig {
   background?: string;
 }
 
+// tslint:disable-next-line: no-empty-interface
+export interface LovelaceBadgeConfig extends EntityConfig {}
```

Comment:
I don't think that the basic lovelace badge config should extend from entity config. It will be possible to have non-entity related badges.